### ISSUE TITLE
LDAPs

### ISSUE THREAD

**Issue by FRodero** [DeviceFarmer/stf#132]
Hello,

Is it possible to configure STF with LDAPs? I need check access with LDAP over SSL.

Thanks a lot.

**Comment by denis99999** [comment]
@FRodero , yes it is, look at DEPLOYMENT.md file for some advices and the online help using the STF CLI: `stf auth-ldap --help`
